SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are applied to the latest MINOR version of Jekyll, and the version used by GitHub Pages, v3.9.x.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 4.2.x   | :white_check_mark: |
+| 3.9.x   | :white_check_mark: |
+| < 3.8.x | :x:                |
+
+## Reporting a Vulnerability
+
+Please report vulnerabilities by sending an email to security@jekyllrb.com with the following information:
+
+1. A description of the vulnerability
+1. Reproduction steps and/or a sample site (share a private repo to the Jekyll Core Team and @parkr)
```

Comment:
List-item no. out-of-order (not important, but still..)

```suggestion
1. Reproduction steps and/or a sample site (share a private repo to the Jekyll Security Team)
```

Reply:
We should enumerate the members here (since repo sharing is on a per-GitHub-user basis) or enumerate the members elsewhere.